S21/FaizanSyed95/Language_test.py

```diff
@@ -0,0 +1,60 @@
+import csv 
+
+Vowels=['A','E','I','O','U','Á','É','Í','Ó','Ú']
+Greek=['Π','Ό','Λ','Η','Σ']
+Greek_again=['π','ό','λ','η','ς']
+
+
+def Word_check(lang , word):
+    LG=list(lang)
+    
```

Comment:
Maybe there's a specific reason that the language code string is split into two conditionals, but I think they could be combined into one. For example, use 'ga', instead of 'g' and 'a'.

Reply:
The reason was to save the code from different errors. Checking each string/char separately has a low risk of bugs in the code. Also, if there are other characters after this it will only detect the 1st two to give an output.

Reply:
Language codes (ISO 639-1) are specific to each individual language. I understand your reason. I am just suggesting using two characters as one string.

Reply:
Yes, they are, however, I don't think it would matter to use them as a string or separate characters. However, thank you for suggesting that I will look that up in the future, it might be less time-consuming.